docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -0,0 +1,58 @@
+# [Upgrade from v7.0.0-beta4 to Unreleased](https://github.com/shopsys/shopsys/compare/v7.0.0-beta4...HEAD)
+
+This guide contain instructions to upgrade from version 7.0.0-beta4 to Unreleased.
+Before you start, don't forget to take a look at [general instructions](/UPGRADE.md) about upgrading.
+There you can find links to upgrade notes for other versions too.
+
+## [shopsys/migrations]
```

Comment:
shouldn't this section be placed below /framework?

Reply:
Indeed. I moved it and update instructions about writing upgrade.